pol-core/pol/base/position.cpp

```diff
@@ -161,6 +156,12 @@ Pos2d Pos2d::max( const Pos2d& v ) const
   return Pos2d( std::max( _x, v._x ), std::max( _y, v._y ) );
 }
 
+u16 Pos2d::clip_u16( int v )
```

Comment:
isn't this just a helper function? why is it in the pos2d class? :D

Reply:
Good point, working on it :)

Reply:
Done